packages/db/src/test-utils.ts

```diff
@@ -146,9 +147,20 @@ export const fakeAsyncOperationRecordFactory = (
 
 export const fakePdrRecordFactory = (
   params: Partial<PostgresPdr>
-) => ({
+): Partial<PostgresPdr> => ({
```

Comment:
Nice.

Reply:
I don't remember what I was doing, but there was some odd output from this file that indicated this factory was behaving differently from the others

Reply:
Always good to type our returns 👍